prov/efa/src/efa_mr.c

```diff
@@ -557,6 +557,71 @@ static struct ibv_mr *efa_mr_reg_ibv_mr(struct efa_mr *efa_mr, struct fi_mr_attr
 			mr_attr->mr_iov->iov_len, access);
 }
 
+static struct fi_mr_attr *
+dup_mr_attr(const struct fi_mr_attr *attr, uint64_t flags)
```

Comment:
Please make the current implementation available for re-use, rather than copying it here.  When exposing it, add the ofi prefix.